frontend/src/components/AboutUs.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+const AboutUs = () => {
+  return (
+    <>
+      <h1>About Us</h1>
+      <h2>Learn more about the team</h2>
+
+      <h3>Data Science</h3>
+      <section>Marc Hendel</section>
+      <section>Juliana Gomes</section>
+
+      <h3>UX Design</h3>
+      <section>Ebunoluwa Meroyi</section>
+      <section>Berenika Godlewska</section>
+      <section>Ngoc Anh Doan Thi</section>
+
+      <h3>Web Development</h3>
+      <section>Stephanie Mennear</section>
+      <section>Timea Varju</section>
+
+      <h2>About TechLabs</h2>
```

Comment:
`section` should be wrapping each heading and the names after it until the next heading, and the names should be wrapped by `p` tags.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section